src/pal/tests/palsuite/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@ project(PALTESTSUITE)
 
 include_directories(${PALTESTSUITE_SOURCE_DIR}/common)
 
+# All test will link against these libraries:
+list(APPEND COMMON_TEST_LIBRARIES pthread)
```

Comment:
single set command would be less chatty:
```cmake
set(COMMON_TEST_LIBRARIES
    pthread
    m
    coreclrpal)
```
However, since the pthread will be conditionalized in one of the next changes, it probably doesn't matter.